KmsClient.js

```diff
@@ -4,9 +4,10 @@
 import base64url from 'base64url-universal';
 import {httpClient, DEFAULT_HEADERS} from '@digitalbazaar/http-client';
 import {signCapabilityInvocation} from 'http-signature-zcap-invoke';
-import {SECURITY_CONTEXT_V2_URL} from 'jsonld-signatures';
-
+import * as sec from 'security-context';
 import {cryptoLd} from './cryptoLd.js';
+const securityContext = sec.constants.SECURITY_CONTEXT_V2_URL;
```

Comment:
If not using SECURITY_CONTEXT_V2_URL, I'd still like to see some all cap var name that has the term `URL` in it to specify that it's a URL and not a context document or other.